### PR TITLE
React & Redux group project - Space Travelers' Hub

### PR DESCRIPTION
**Pull request by Peter-Dumbari**
## In this PR:
- We created a new repository and created `dev` branch which is the default branch 
- We have another branch called `development`  which serves as the development branch.
- we created other features branches and work according to the Kanban board, merging them at the end of the project
- We ran jest test 
- We ran linter validation 
- we created a pull request

### CODE REVIEW

File: src/Test/Mission.test.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+test('handles join and leave mission', () => {
+  const store = mockStore({ mission: { mission: [] } });
+
+  render(
+    <Provider store={store}>
+      <Mission />
+    </Provider>,
+  );
+});
```
Comment (Abenezer-Tilahun) [inline src/Test/Mission.test.js:27]: - [x] As stated in the project requirement Kindly make sure that to **add test for all your components and functions.**  and make sure that all of your test cases are pass but in the case of yours your test case for mission component is not passed Kindly Fix It.
test case

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @Peter-Dumbari  @gergitokusse   , 

Great job for implemented  the some of the Project Requirement , but there are some changes that need to be implemented in order to proceed to the next project 👍

## Highlights 👏🏻
✔️ No Linter errors
✔️ Correct Git -Flow
✔️ Good commit messages
✔️ Professional README

almost

## Required Changes ♻️
_Check the comments below the review._

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Test/Mission.test.js (hunk 1 above)
Comment (Peter-Dumbari) [reply src/Test/Mission.test.js:27]: @Abenezer-Tilahun, If I remove the `css` link, the test pass successfully
Screen Shot 2023-06-30 at 4 40 53 PM

Comment (FrederickMih) [inline src/Test/Mission.test.js:27]: - [ ] The two tests written are running successfully :+1: As requested by the previous reviewer, you'll be required to add more tests for the functionalities of your app. You wrote a single test to render the Mission component while you're equally required to write add more tests for more components and some pure functions in the redux store.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
## Hi :raising_hand: :raising_hand: @Team, 


Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

- Unit tests are running successfully :+1: 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Test/Rocket.test.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+test('renders Rockets component', () => {
+  const store = mockStore({ rockets: { rockets: [] } });
+
+  render(
+    <Provider store={store}>
+      <Rockets />
+    </Provider>,
+  );
+});
```
Comment (OyePriscilla) [inline src/Test/Rocket.test.js]: - [ ] Kudos for adding test as suggested earlier, 👍 can you please add tests for functions as well. ℹ️ This [link](https://redux.js.org/usage/writing-tests#unit-testing-individual-functions) might be of help. ℹ️ Kindly read through this header in particular **`Unit Testing Individual Functions.`**

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @Peter-Dumbari and @gergitokusse 👍, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

so-close-so-close-seth-meyers

 ## Highlights
Implemented test for `rocket `component 👍 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by binhussen — APPROVED** [review]
Hi @Peter-Dumbari and @gergitokusse , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- Tested using Testing Library ✔️
- Well Structured Code ✔️
- Descriptive pull request ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**